### ISSUE TITLE
fullspeedwhenloading doesn't respect active machine

### ISSUE THREAD

**Issue by MBilderbeek** [openMSX/openMSX#982]
Original comment: "Wanneer je de setfullspeedwhenloading aanzet, zet openMSX feitelijk de max Throttle aan en weer uit als hij klaar is met laden. Echter, wanneer je tijdens het laden (van b.v. een *.CAS die best even tijd kosten, zelfs met de Throttle aan) even naar een andere machine tab gaat blijft de Throttle aanstaan. De machine die aan het laden is wordt wel gepauzeerd maar openMSX “vergeet” ook even de Throttle weer uit te zetten zodat de current tab op normale snelheid loopt."

Summary: when fullspeedwhenloading is used in combination with tabbed MSXing, things go wrong: if you switch to another tab while fullspeedwhenloading is active, openMSX continues in full throttle also for the other tab, while the first machine (for which full throttle was enabled) is paused.

Fullspeedwhenloading is not 'current machine aware'.

**Comment by MBilderbeek** [comment]
I guess ThrottleManager should move from global to machine specific. Wouter, any ideas on how to design this?

**Comment by MBilderbeek** [comment]
@m9710797 please see 7+ year old question above :)

**Comment by m9710797** [comment]
I agree, `ThrottleManager` should be per-machine instance. We should change this.
But IMHO this is low priority, personally I think there are other more important items. But I can help if someone volunteers to tackle this.
